backend/private-graph/graph/schema.graphqls

```diff
@@ -1071,6 +1085,24 @@ type SessionAlert {
 	disabled: Boolean!
 }
 
+type LogAlert {
+	id: ID!
+	updated_at: Timestamp!
+	Name: String!
+	ChannelsToNotify: [SanitizedSlackChannel!]!
```

Comment:
could we rename this to SlackChannelsToNotify?

Reply:
I wanted to keep it `ChannelsToNotify` just because that's the name on the base `Alert` model